client/src/actions/index.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import {someUrl} from "../constants/urls"
+
+export const ACTION_TYPES = {
```

Comment:
Can we put all the action types into a constants file?

Reply:
Good idea!